File: src/main/java/org/codinjutsu/tools/nosql/commons/view/add/NullFieldWrapper.java
package org.codinjutsu.tools.nosql.commons.view.add;

import org.codinjutsu.tools.nosql.commons.model.internal.layer.DatabaseElement;

import javax.swing.JLabel;

public class NullFieldWrapper extends TextFieldWrapper<JLabel> {

    public NullFieldWrapper() {
        super(new JLabel("null"));
    }

    @Override
    public DatabaseElement getValue() {
        return null;
    }

    @Override
    public void reset() {
    }
}

File: src/main/java/org/codinjutsu/tools/nosql/commons/view/add/NullFieldWrapper.kt
package org.codinjutsu.tools.nosql.commons.view.add

import org.codinjutsu.tools.nosql.commons.model.internal.layer.DatabaseElement

import javax.swing.JLabel

internal class NullFieldWrapper : AbstractTextFieldWrapper<JLabel>(JLabel("null")) {

    override fun getValue(): DatabaseElement? = null

    override fun reset() {}
}
